gradle/gradle-mvn-push.gradle

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
Using the standard gradle-mvn-push script, also defined in our OSS template.

Reply:
If you ever add kotlin in the future, someone contributed dokka support to this in AutoDispose. Just FYI